packages/flutter/lib/src/material/chip.dart

```diff
@@ -441,6 +441,18 @@ abstract interface class DisabledChipAttributes {
   ///
   /// It defaults to [Colors.black38].
   Color? get disabledColor;
+
+  /// The color used for the chip's background to indicate that it is selected
+  /// and disabled.
+  ///
+  /// The chip is disabled when [isEnabled] is false, or all three of
+  /// [SelectableChipAttributes.onSelected], [TappableChipAttributes.onPressed],
+  /// and [DeletableChipAttributes.onDeleted] are null.
+  ///
+  /// The chip is selected when [SelectableChipAttributes.selected] is true.
+  ///
+  /// It defaults to [Colors.black38].
+  Color? get disabledSelectedColor;
```

Comment:
Did you consider just supporting a MaterialStateColor value for selectedColor? Or just adding a color property with a `MaterialStateProperty<Color?>` value?

Reply:
> Did you consider just supporting a MaterialStateColor value for selectedColor? 

I didn't consider it since it will be inconsistent to just have one color that supports `MaterialStateColor`.

> Or just adding a color property with a MaterialStateProperty<Color?>? value?

This is something I wanted to do but I wasn't sure. I can add a regular `color` property as you suggested. :)